.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: node_js
 
-node_js:
```

Comment:
This is the difference between what you and I did, I kept node 8 here and it blew up.

Reply:
It worked when that was still in there too fwiw, I only took it out after @vladikoff mentioned something in the auth server PR.

I thought maybe you forgot to update shrinkwrap and that overrode the optional dependencies or something?